module/data/item/templates/physical-item.mjs

```diff
@@ -19,9 +19,14 @@ export default class PhysicalItemTemplate extends foundry.abstract.DataModel {
       weight: new foundry.data.fields.NumberField({
         required: true, nullable: false, initial: 0, min: 0, label: "DND5E.Weight"
       }),
-      price: new foundry.data.fields.NumberField({
-        required: true, nullable: false, initial: 0, min: 0, label: "DND5E.Price"
-      }),
+      price: new foundry.data.fields.SchemaField({
+        value: new foundry.data.fields.NumberField({
+          required: true, nullable: false, initial: 0, min: 0, label: "DND5E.Price"
+        }),
+        currency: new foundry.data.fields.StringField({
```

Comment:
Slight preference for 'denomination', or 'denom' instead. What do you think?

Reply:
Probably a more accurate term.